chain/src/txhashset.rs

```diff
@@ -1150,9 +1150,11 @@ fn input_pos_to_rewind(
 		// I/O should be minimized or eliminated here for most
 		// rewind scenarios.
 		let current_header = commit_index.get_block_header(&current)?;
-		let input_bitmap = commit_index.get_block_input_bitmap(&current)?;
-		found = input_bitmap.0;
-		bitmap.or_inplace(&input_bitmap.1);
+		let input_bitmap_res = commit_index.get_block_input_bitmap(&current);
```

Comment:
Note this is wrong.. in fast sync case `get_block_input_bitmap` is returning an error due to no block or bitmap existing in the DB.